api/users/serializers.py

```diff
@@ -1,6 +1,30 @@
 from rest_framework import serializers as ser
-
 from api.base.serializers import JSONAPISerializer, LinksField, Link
+from website.models import User
+
+
+class ProfileDictSerializer(ser.ListField):
+    start_year = ser.CharField(allow_blank=True, allow_null=True)
```

Comment:
This doesn't seem to be working, when I GET my user info, I see the camel-cased fields.

![django_rest_framework](https://cloud.githubusercontent.com/assets/2379650/9206992/ebad41de-4038-11e5-8c6b-4eb08c5ad811.png)